components/Button.js

```diff
@@ -0,0 +1,18 @@
+// create Button component
+import React, { Component } from 'react';
+// import view for creating and stylesheet for changing the appearance later
+import { View, StyleSheet } from 'react-native';
+
+function TBDmethod(){
+   // do whatever we want button to do     
```

Comment:
we could also pass the function/method that the button does as a prop to make the button more reusable if we have buttons doing different things